solutions/banking/watson-scripts/assistant-destroy.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+project_name="gen-ai-rag-sample-app-assistant"
+token=$(curl -fLsS -X POST 'https://iam.cloud.ibm.com/identity/token' -H 'Content-Type: application/x-www-form-urlencoded' -d "grant_type=urn:ibm:params:oauth:grant-type:apikey&apikey=$API_KEY" | jq -r '.access_token')
+
+ASSISTANT_ID=$(curl --retry 3 -fLsS -X GET --location "$WATSON_ASSISTANT_URL/v2/assistants?version=2023-06-15" \
```

Comment:
I think the ASSISTANT_ID value should come from the data the shell-script feeds to the script - the assistant ID should be stored in the state along with other attributes.
You could do it with something like (depending on what's stored in the state)
```
IN=$(cat)
assistant_id=$(echo $IN | jq -r .id)
```